.github/workflows/android-ci.yml

```diff
@@ -58,6 +58,9 @@ jobs:
           submodules: recursive
           fetch-depth: 0
       
+      - name: Setup Android SDK and accept licenses
+        uses: android-actions/setup-android@v2
+            
```

Comment:
We use a docker container to have a reproducible environment. Maybe we should actually change the container? I don't know how this stuff works, but if we use the container, I think we should not change the environment inside it...